guaranteed-message-delivery/request_handler.bal

```diff
@@ -0,0 +1,42 @@
+import ballerina/http;
+import ballerinax/rabbitmq;
+
+type Patient record {|
+    string name;
+    string dob;
+    string ssn;
+    string address;
+    string phone;
+    string email;
+|};
+
+type ReservationRequest record {|
+    Patient patient;
+    string doctor;
+    string hospital_id;
+    string hospital;
+    string appointment_date;
+|};
+
+configurable string queueName = "ReservationQueue";
+configurable string exchangeName = "ReservationExchange";
```

Comment:
Do we need this `Exchange` name? IMO using the default exchange would be good these type of samples.